code/game/jobs/faction/hephaestus.dm

```diff
@@ -69,18 +69,21 @@
 	name = "Engineer - Hephaestus"
 
 	uniform = /obj/item/clothing/under/rank/engineer/heph
+	head = /obj/item/clothing/head/hardhat/red
```

Comment:
No. Same issue as said before, head items fuck over loadout preview.

Reply:
the default engineer parent spawns with a hardhat as well